tests/documentation/test_files/DocumentationStyleBear/bad_file4.py.test.correct

```diff
@@ -0,0 +1,12 @@
+def myfunc(a, b, x):
+    """
+
+    :param a:
+        slope
+    :param b:
+        offset
+    :param x:
+    :return:
+    """
+
```

Comment:
@jayvdb Oh it's the other way around :O . What I and Sanket discussed while before the GSoC started that we should have a "blank line before and after" which I guess you wanted to portray as voilation. Tell me if im wrong? Though I will just take a quick look at PEP - 257.